src/lib/Hydra/Controller/Root.pm

```diff
@@ -168,7 +168,7 @@ sub queue_runner_status_GET {
     my ($self, $c) = @_;
 
     #my $status = from_json($c->model('DB::SystemStatus')->find('queue-runner')->status);
-    my $status = from_json(`hydra-queue-runner --status`);
+    my $status = decode_json(`hydra-queue-runner --status`);
```

Comment:
Pretending I didn't see this backtick business at the moment.

Reply:
also the commented out line above?

Reply:
many terrors